### ISSUE TITLE
TrailingWhitespace and DuplicateProperty bug

### ISSUE THREAD

**Issue by geta6** [rossPatton/stylint#207]
When linting with `trailingWhitespace: true`, `duplicates: true` or `valid: true`, you get warns against expectations.

```stylus
$icon-selector = '.icon'

.label, {$icon-selector} // warn: trailing white space
  font-size 12px

button
  .label
    font-size 11px
  {$icon-selector}  // warn: duplicate property or selector
    font-size 10px

{$icon-selector}-user // warn: property is not valid
  color blue
```

I think this code is probably the cause, how do you think?

src/utils/trimline.js

```javascript
var trimLine = function( line ) {
  :
  // strip interpolated variables
  return line.replace( /({\S+})/, '' ) // /({.+})
}
```

- stylint 1.2.4
- OSX 10.11

**Comment by rossPatton** [comment]
your assumption is most likely correct here, thx for letting me know

**Comment by rossPatton** [comment]
so fixing the 1st and 3rd issues appear to be pretty straightforward, and will be in the next release.

not able to reproduce the 2nd issue quite yet

**Comment by rossPatton** [comment]
1st and 3rd should be fixed in 1.3.0, closing because i haven't been able to reproduce issue 2
